src/client/components/ConsentCardOnboardingEmails.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FunctionComponent } from 'react';
+import { SerializedStyles, css } from '@emotion/react';
+import {
+  space,
+  neutral,
+  from,
+  textSans,
+  until,
+} from '@guardian/source-foundations';
+import { ToggleSwitchInput } from './ToggleSwitchInput';
+import { greyBorderTop, text } from '@/client/styles/Consents';
+import { topMargin } from '@/client/styles/Shared';
+
+const containerStyles = css`
+  display: flex;
+  flex-direction: column;
+  border: ${neutral[0]} 3px dashed;
+  border-radius: 12px;
+  margin-bottom: ${space[3]}px;
+  padding: ${space[2]}px;
+
+  ${from.tablet} {
+    padding: ${space[2]}px ${space[3]}px;
+  }
+
+  :not(:last-of-type) {
+    margin-bottom: ${space[5]}px;
+  }
+`;
+
+const toggleSwitchAlignment = css`
+  justify-content: space-between;
+  span {
+    align-self: flex-start;
+    margin-top: ${space[2]}px;
+  }
+`;
+
+const labelStyles = css`
+  ${textSans.small()}
+  font-weight: bold;
+  span {
+    margin-top: 2px !important;
+  }
+
+  ${until.mobileLandscape} {
+    span {
+      margin-left: ${space[5]}px;
+    }
+  }
+`;
+
+const switchRow = css`
+  border: 0;
+  padding: 0;
+  ${topMargin}
+`;
+
+interface ConsentCardOnboardingEmailProps {
+  id: string;
+  defaultChecked?: boolean;
+  cssOverrides?: SerializedStyles;
+}
+
+export const ConsentCardOnboarding: FunctionComponent<
+  ConsentCardOnboardingEmailProps
+> = ({ id, defaultChecked, cssOverrides }) => {
+  return (
+    <article css={[containerStyles, cssOverrides]}>
+      <p css={[text]}>
+        As part of having an account, you’ll receive weekly emails for a month
+        to help you get the most out of the Guardian.
+      </p>
+      <fieldset css={[switchRow, greyBorderTop]}>
+        {/* Hidden input required to capture unsubscribe events */}
```

Comment:
This emulates how unsubscribe events are posted by the form in the existing newsletter component.  The Data page that has been fully converted to toggle switches works differently, but as the switch is a checkbox masquerading as a toggle, I have gone with this.